mailer.js

```diff
@@ -268,15 +268,22 @@ module.exports = function (log) {
 
     var links = this._generateLinks(this.verificationUrl, message.email, query, templateName)
 
+    var headers = {
```

Comment:
Nit if you want, I guess we prefer `let` now over `var` wherever possible. If I didn't have the question about `flowBeginTime`, I would have merged regardless, so however you like.